app/src/main/admin/components/ModellingGroups/Models/ModelMetaTable.tsx

```diff
@@ -110,82 +115,88 @@ export class ModelMetaTableComponent extends React.Component<ModelMetaProps, Sta
 
     render() {
 
+        const obsolete = this.props.obsoleteModels;
+
+        if (obsolete && this.state.data.length == 0) {
+            return "";
+        }
+
         return <div>
+            {!obsolete || <p>The following obsolete models were also found.</p>}
```

Comment:
Is there a reason to use inverted logic here? If not I think `{obsolete && <p></p>}` is easier to read.